<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaPrincipal</class>
 <widget class="QMainWindow" name="TelaPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>573</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Grafos</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/BannerofPeace.png</normaloff>:/BannerofPeace.png</iconset>
  </property>
  <property name="locale">
   <locale language="Portuguese" country="Brazil"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBoxEntradas">
      <property name="title">
       <string>Entradas</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Número de vértices</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="inNumeroVertices">
           <property name="statusTip">
            <string>Define o número de vértices do grafo</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="botaoOkNumeroVertices">
           <property name="statusTip">
            <string>Confirma o número de vértices do grafo</string>
           </property>
           <property name="text">
            <string>Ok</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxLigacoes">
      <property name="title">
       <string>Ligações</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Origem</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="inOrigem">
           <property name="statusTip">
            <string>Defina o vértice de origem da ligação</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Tipo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="botaoTipoLigacao">
           <property name="statusTip">
            <string>Defina o tipo da ligação::: Arco----&gt; | Aresta -----</string>
           </property>
           <property name="text">
            <string>----&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Destino</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="inDestino">
           <property name="statusTip">
            <string>Defina o vértice de destino da ligação</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Peso</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="inPeso">
           <property name="statusTip">
            <string>Define a distância entre os vértices da ligação</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item alignment="Qt::AlignBottom">
        <widget class="QPushButton" name="botaoAdicionarLigacao">
         <property name="statusTip">
          <string>Adiciona uma ligação entre vértices no grafo</string>
         </property>
         <property name="text">
          <string>Adicionar</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>202</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxOpcoes">
      <property name="title">
       <string>Opções</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Vértice Inicial:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="inVerticeInicial">
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="botaoConexidade">
           <property name="text">
            <string>Conexidade</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="botaoCaixeiroViajante">
           <property name="statusTip">
            <string>Caixeiro viajante, heurística: vértice adjacente mais próximo</string>
           </property>
           <property name="text">
            <string>Caixeiro Viajante</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="botaoTestarPlanaridade">
           <property name="toolTip">
            <string>Testa a planaridade do grafo</string>
           </property>
           <property name="text">
            <string>Testar Planaridade</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="botaoColorirGrafo">
           <property name="toolTip">
            <string>Colorir grafo</string>
           </property>
           <property name="styleSheet">
            <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 255, 0, 255), stop:0.333 rgba(0, 255, 0, 255), stop:0.5 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(255, 0, 255, 255), stop:1 rgba(255, 0, 0, 255))</string>
           </property>
           <property name="text">
            <string>Colorir</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTabWidget" name="abas">
         <property name="tabPosition">
          <enum>QTabWidget::North</enum>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::Rounded</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="abaLigacoes">
          <attribute name="title">
           <string>Ligações</string>
          </attribute>
          <attribute name="toolTip">
           <string>Aba que indica as ligações do grafo</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QTableWidget" name="tabelaLigacoes">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>70</number>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Origem</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Tipo</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Destino</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Peso</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="abaMatrizAdjacencia">
          <attribute name="title">
           <string>Matriz Adjacência</string>
          </attribute>
          <attribute name="toolTip">
           <string>Aba que exibe a matriz de adjacência do grafo</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QTableWidget" name="tabelaAdjacencia">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="abaGrafo">
          <attribute name="title">
           <string>Grafo</string>
          </attribute>
          <attribute name="toolTip">
           <string>Aba que exibe o desenho do grafo</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QGraphicsView" name="areaDesenhoGrafo">
             <property name="renderHints">
              <set>QPainter::Antialiasing</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>405</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="statusTip">
     <string>Criar uma nova configuração | Sair do aplicativo</string>
    </property>
    <property name="locale">
     <locale language="Portuguese" country="Brazil"/>
    </property>
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionNovo"/>
    <addaction name="separator"/>
    <addaction name="actionSair"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNovo">
   <property name="text">
    <string>Novo</string>
   </property>
   <property name="statusTip">
    <string>Resetar configurações</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
   <property name="statusTip">
    <string>Sair do programa</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
